<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reporter</class>
 <widget class="QMainWindow" name="Reporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reporter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="reportGenerator">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GENERAR REPORTE</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>        FECHA INICIO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="startDate"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>      FECHA TERMINO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="finalDate"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="folderPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectFolder">
       <property name="text">
        <string>FOLDER</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
